app/src/main/java/com/example/mydigitalcloset/OutfitCreationActivity.java

```diff
@@ -252,14 +163,16 @@ public void onClick(View view) {
                 finish();
             }
         });
-        /*binding.settingsButton.setOnClickListener(new View.OnClickListener() {
+        binding.settingsButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                Intent intent = new Intent(getApplicationContext(), ?.class);
+                Intent intent = new Intent(getApplicationContext(), SettingsPage.class);
```

Comment:
Make sure that the settings page class matches up with the settings page that we have. And being able to click on different pages to get to the settings page.

Reply:
double checked settings button functionality and name, everything matches up